apps/amakrushi/lang/en.json

```diff
@@ -22,6 +22,7 @@
     "label.new_chat":"New chat",
     "label.confirm_delete":"Are you sure you want to delete this conversation?",
     "label.no_history":"No Chats",
+    "label.translate":"Translate",
```

Comment:
Should we rename this to transliterate? It has to be `transliterate` everywhere.

Reply:
done